src/extension/background-script/connectors/commando.ts

```diff
@@ -303,10 +306,16 @@ export default class Commando implements Connector {
       params: {},
       rune: this.config.rune,
     })) as CommandoListFundsResponse;
-    const lnBalance = response.channels.reduce(
-      (balance, channel) => balance + channel.channel_sat,
-      0
-    );
+    const lnBalance = response.channels
```

Comment:
https://github.com/ElementsProject/cln-application/blob/main/apps/frontend/src/store/AppContext.tsx#L139

Reply:
maybe add it as comment?